### PR TITLE
Feature/site public

### PR DESCRIPTION
**Pull request by krazemon**

### CODE REVIEW

File: js/components/CategoryPage.jsx [hunk 1]
```diff
@@ -65,7 +63,7 @@ var CategoryPage = React.createClass({
     var categoryPageHeader = '';
     var trendingTable = '';
     var categoryDashboard = '';
-    if (this.state.category && this.state.currentUser) {
+    if (this.state.category) {
```
Comment (ritmatter) [inline js/components/CategoryPage.jsx]: Could you put a comment here that we do actually check for a current user?

File: js/components/HomePage.jsx [hunk 2]
```diff
@@ -65,14 +68,14 @@ var HomePage = React.createClass({
...
-              <ProfileQuickView />
+              {facebookInvite}
+              {profileQuickView}
             </div>
           </div>
           <div className="col-md-9">
             {categoriesCards}
             <div className="main-dashboard">
-              {dashboard}
+              <Dashboard        currentUser={this.state.currentUser} />;
```
Comment (ritmatter) [inline js/components/HomePage.jsx]: lots of spaces?

File: js/components/ProfilePage.jsx [hunk 3]
```diff
@@ -69,11 +66,12 @@ var ProfilePage = React.createClass({
     var portfolio = '' ;
     var profileBox = '';
 
-    if (this.state.user && this.state.currentUser) {
-      var isSelf = this.state.currentUser._id === this.props.params.userId;
-
+    if (this.state.user) {
+      var isSelf = this.state.currentUser && this.state.currentUser._id === this.props.params.userId;
+      console.log(!this.state.currentUser);
```
Comment (ritmatter) [inline js/components/ProfilePage.jsx]: extra console statement?

File: js/components/PublicUserBox.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var auth = require('../auth.jsx');
+var React = require('react');
+var Router = require('react-router');
+var Link = Router.Link;
+var strings = require('../lib/strings_utils.js');
+
+var PublicUserBox = React.createClass({
```
Comment (ritmatter) [inline js/components/PublicUserBox.jsx]: Extraneous file?

File: server.js [hunk 5]
```diff
@@ -4,9 +4,9 @@ require('newrelic');
 var bodyParser = require('body-parser');
 var cookieSession = require('cookie-session');
 var express = require('express');
+var livereload = require('express-livereload');
```
Comment (ritmatter) [inline server.js]: do we want this in here yet?

**Comment by ritmatter** [comment]
LGTM once you address my questions

**Comment by krazemon** [comment]
LGTM

**Pull request merged**